### PR TITLE
Fixed https:// in the ServerURL 

### PR DESCRIPTION
**Pull request by saketkumar95**
@borisyankov  Please review

Whenever there is no url in the input box and pressing enter `https://https:` is appended. 

Before : 
screen shot 2017-06-23 at 10 15 04 am

pressing enter only appends `https://`
After :
screen shot 2017-06-23 at 10 16 09 am

### CODE REVIEW

File: src/start/RealmScreen.js [hunk 1]
```diff
@@ -90,7 +83,7 @@ class RealmScreen extends React.Component {
               autoCapitalize="none"
               placeholder="Server URL"
               defaultValue={realm}
-              onChangeText={value => this.setState({ realm: value })}
+              onChangeText={value => this.setState({ realm: fixRealmUrl(value) })}
```
Comment (vishwesh3) [inline src/start/RealmScreen.js]: surprising behaviour 
ezgif com-video-to-gif-7

Comment (saketkumar95) [inline src/start/RealmScreen.js]: Yeah , I know this. Don't know why it's happening. That's why I asked if it's a good way to do it?
Apart from this, anything you write starts with `https://` Just this thing is lagging.

Comment (borisyankov) [inline src/start/RealmScreen.js]: Setting state on each keystroke is neither performant nor logically correct.

**Comment by borisyankov** [comment]
@saketkumar95 the current implementation is not working correctly (and slows down the input)
Please, reconsider your approach to the issue.

**Comment by saketkumar95** [comment]
@borisyankov Yeah I know, currently trying to solve, that's why I kept this PR open. I'm working more for android.  Any other issue you know about this?

**Comment by saketkumar95** [comment]
@borisyankov Done a bit change. Don't know why checks are failing?

**Comment by borisyankov** [comment]
Jest is failing one of the tests.

This latest code is reversing a feature that adds 'https' and '/' to the URL. Not a solution.

Closing this PR, reopen if you solve without removing the code.

**Pull request closed without merging**